ec2/variables.tf

```diff
@@ -32,3 +32,7 @@ variable "subnet_id" {}
 variable "public_key" {
```

Comment:
Maybe put a line break between variables to make it more readable

Reply:
The variable below isn't needed so I've just left the line break at the end.